add rwc archetypes lib description

rwcarlsen opened the pull request.

mbmcgarry commented on source/user/index.rst:
```diff
@@ -117,6 +117,47 @@ Third-Party Archetypes
       or at Random timesteps, and quantity requested can be varied using a
       Gaussian distribution function.
  
+**rwc**
```
perhaps heading title could be in caps just for asthetics (ie: RWC)

mbmcgarry commented on source/user/index.rst:
```diff
@@ -117,6 +117,47 @@ Third-Party Archetypes
+    see if its chosen deployments were not "good" and adjusts its deployment
+    decisions accordingly. This archetype currently requires
+    rwcarlsen/cyclus#restart branch of Cyclus to work.
+
```
I think it looks good with a whitespace between each archetype, but this is inconsistent with the mbmore descriptions above.  Any chance you want to throw a whitespace in the mbmore examples to make it all consistent?

mbmcgarry replied:
Also inconsistent between the sections is bullet-ing for each listed archetype

mbmcgarry commented:
Can you reorder the archetypes with some logic - perhaps alphabetical ?

mbmcgarry commented:
General question ( @rwcarlsen @scopatz @gonuke @opotowsky @Baaaaam ): As we add 3rd party archetypes, do we want to require any quality control (unit tests, code review, formatting etc?)

opotowsky commented:
The general rule is it's up to the user to defend their work if they use certain archetypes, but I can see why we would want to have a standard for what we feature on the website.

scopatz commented:
I agree with @opotowsky - 3rd party libs are by definition outside of the cyclus org's qc.

rwcarlsen commented:
I think we don't worry about it - especially at these earlier stages in
archetype community growth.  When the list becomes sufficiently large,
perhaps we can become more selective.  Maybe it would be worth throwing in
a one-sentence disclaimer at the top of the "third-party" archetypes
section though.

On Thu, Apr 21, 2016 at 1:22 PM, Anthony Scopatz <notifications@github.com>
wrote:

> I agree with @opotowsky <https://github.com/opotowsky> - 3rd party libs
> are by definition outside of the cyclus org's qc.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus.github.com/pull/191#issuecomment-213048700>
>

rwcarlsen commented:
I added a disclaimer to the 3rd party section and cleaned up formatting in @mbmcgarry and rwc archetypes sub-sections.  @mbmcgarry - the order is explicit - it was chosen in order of awesomeness - or in the order I would want a user to read.  But I don't feel super strongly about it - I'd be happy to reorder it as long as the institution is first.

mbmcgarry commented:
@rwcarlsen while I do appreciate the urge to order by awesomeness, I imagine as more archetypes are added it may become difficult to appropriately rank them by awesome. I'd prefer something along the lines of alphabetical within Region->Insititution->Facility or similar

rwcarlsen commented:
Fair enough - re-ordered.

On Fri, Apr 22, 2016 at 4:05 PM, mbmcgarry <notifications@github.com> wrote:

> @rwcarlsen <https://github.com/rwcarlsen> while I do appreciate the urge
> to order by awesomeness, I imagine as more archetypes are added it may
> become difficult to appropriately rank them by awesome. I'd prefer
> something along the lines of alphabetical within
> Region->Insititution->Facility or similar
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus.github.com/pull/191#issuecomment-213588633>
>

mbmcgarry commented:
looks great, and thanks for standardizing the formatting for the entire Third-Party Archetypes section!
